lisa/sut_orchestrator/azure/tools.py

```diff
@@ -55,3 +58,63 @@ def get_generation(self) -> str:
         else:
             generation = "1"
         return generation
+
+
+class LisDriver(Tool):
+    """
+    This is a virtual tool to detect/install LIS (Linus Integration Services) drivers.
```

Comment:
LIS == Linux Integration Services

Reply:
> LIS == Linux Integration Services

Thanks for noting this. Lili is taking care of this in her timesync PR -

https://github.com/microsoft/lisa/pull/1345/files#diff-3d73998ffe81f214543e8797e2f8e6fbeb9ce2e5d9f1a3f9feceb6d68dcb7dc6